docs/DeveloperManuals/Project.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Project"
+sidebar_position: 5
+description: >
+  Project is an entity object that can be used for Dora to associate pr to deployment/issue.
```

Comment:
`Project` is an object that can be used for DORA to associate `changes(commits)`, with `deployments` and `issues`.